pyqtgraph/exporters/Matplotlib.py

```diff
@@ -53,28 +53,46 @@ def cleanAxes(self, axl):
                     raise ValueError('Unknown spine location: %s' % loc)
                 # turn off ticks when there is no spine
                 ax.xaxis.set_ticks_position('bottom')
-    
+
     def export(self, fileName=None):
-        
+
         if isinstance(self.item, PlotItem):
             mpw = MatplotlibWindow()
             MatplotlibExporter.windows.append(mpw)
 
             stdFont = 'Arial'
-            
+
             fig = mpw.getFigure()
-            
+
             # get labels from the graphic item
             xlabel = self.item.axes['bottom']['item'].label.toPlainText()
             ylabel = self.item.axes['left']['item'].label.toPlainText()
             title = self.item.titleLabel.text
 
+            # pyqtgraph by default uses scientific notation for large
+            # values. Matplotlib values and label should reflect that too.
+            SIprefix_scale_default = 1.0 # default SI prefix scale value
```

Comment:
I think a more straightforward implementation might loo something like:

```python
scale_x = scale_y = 1.0
if self.items.axes['bottom']['item'].autoSIPrefix: # note no need for == True here
    scale_x = self.item.axes['bottom']['item'].autoSIPrefixScale
if self.items.axes['left']['item'].autoSIPrefix:
    scale_y = self.item.axes['left']['item'].autoSIPrefixScale
```

Then simply apply scale_x/scale_y below without the conditionals.